### ISSUE TITLE
Support for excluding mutators

### ISSUE THREAD

**Issue by christianhujer** [szpak/gradle-pitest-plugin#187]
I would like to be able to specify the mutators like this:

```
pitest {
    // …
    mutators = ['All'] - ['UOI']
    // …
}
```

or, if that is not possible, with something equivalent, like this:

```
pitest {
    // …
    mutators = ['All']
    excludeMutators = ['UOI']
    // …
}
```

Use case: instead of having a long list of all mutators to run, be able to specify a large set of mutators, like `'ALL'`, and then just exclude some from the list, like `['UOI']`, when they produce too many false positives.

**Comment by npathai** [comment]
I will work on it and raise a PR

**Comment by szpak** [comment]
Just remember to test it thoroughly. What possible in unit (or unit-like) tests.

**Comment by npathai** [comment]
@szpak Yep will do that. Will look at all existing tests and add required tests accordingly 👍  Will ask if I have any questions

**Comment by npathai** [comment]
@szpak From what I found out, there is no support for excluding mutators in PIT project. What do you think should be the next step?

**Comment by szpak** [comment]
I didn't need it before, but it seems there is no easy way to exclude mutators (instead of provide just the list of selected mutators to use). It could be implemented in the Gradle plugin alone, but most likely it would be good to have it available in pit-core, so I propose to report that suggestion in the [PIT issue tracker](https://github.com/hcoles/pitest/issues) to see what Henry thinks about that.

**Comment by npathai** [comment]
Apologies for late reply. I will do that.

**Comment by kamesh95** [comment]
Any update here? I wanted to change mutators group to ALL with pitest gradle plugin but I cannot find any property to do that in here. Can you help on this? Thank you

**Comment by szpak** [comment]
@kamesh95 I haven't seen a related issue in the [PIT issue tracker](https://github.com/hcoles/pitest/issues). You might want to create one.

**Comment by npathai** [comment]
@szpak I have raised the feature request in PIT issue tracker.

**Comment by szpak** [comment]
@npathai Could you check it with PIT 1.7.3?

```
pitest {
    pitestVersion = '1.7.3'
    ...
}
```

**Comment by npathai** [comment]
@szpak Sure let me do that. I will raise PR when ready.
